changelog.txt

```diff
@@ -6,3 +6,4 @@
 * Hot reload firestore rules on change.
 * Upgrade archiver dependency to 3.0.0.
 * Update functions init templates to `v3.1.0`
+* Fix emulation of https functions
```

Comment:
Put a period on the end of the sentence.

Reply:
07d718e28f280c2c4ef5103a761ff9da73101a14